test/Microsoft.ML.Tests/Scenarios/Api/AspirationalExamples.cs

```diff
@@ -60,5 +60,98 @@ public void FirstExperienceWithML()
                 PetalLength = 5.1f
             });
         }
+
+
+        [Fact]
+        public void SimpleIrisDescisionTrees()
+        {
+            var env = new TlcEnvironment(new SysRandom(0), verbose: true);
+            string dataPath = "iris-data.txt";
+            // Create reader with specific schema.
+            var dataReader = TextLoader.CreateReader(env, ctx => (
+               Label: ctx.LoadText(0),
+               SepalWidth: ctx.LoadFloat(1),
+               SepalLength: ctx.LoadFloat(2),
+               PetalWidth: ctx.LoadFloat(3),
+               PetalLength: ctx.LoadFloat(4)),
+               dataPath);
+
+            // Load the data into the system.
+            var data = dataReader.Read(dataPath);
+
+            var preprocess = data.Schema.MakeEstimator(row => (
```

Comment:
>data.Schema.MakeEstimator [](start = 29, length = 25)

this should instead be `Estimator.Create(data).Append(row => ...` I guess. Take a look at `Estimator.Create` from Tom

Reply:
While I wrote things that way at your request, I actually am not fond of how the code will look, and would prefer something closer to what @Ivanidzo4ka has done here. I think that if I have any of these schema-bearing objects, I should be able to do something like `.StartPipe` on it or something to get an empty estimator. Perhaps we should open an issue or something to discuss this @Zruty0?

Reply:
let's go over this at today's meeting

---
In reply to: [215056489](https://github.com/dotnet/machinelearning/pull/809#discussion_r215056489) [](ancestors = 215056489)

Reply:
On the discussion, we decided to keep it as `data.MakeEstimator().Append(row=>...)`. I'm not terribly happy with this, but oh well.

---
In reply to: [215063741](https://github.com/dotnet/machinelearning/pull/809#discussion_r215063741) [](ancestors = 215063741,215056489)